2996_reflection/reflection.md

```diff
@@ -1418,12 +1426,12 @@ That is, splicing a constructor behaves like a free function that produces an ob
 
 The splicers described above all take a single object of type `std::meta::info` (described in more detail below).
 However, there are many cases where we don't have a single reflection, we have a range of reflections - and we want to splice them all in one go.
-For that, we need a different form of splicer: a range splicer.
+For that, the predecessor to this paper, [@P1240R0], proposed an additional form of splicer: a range splicer.
 
 Construct the [struct-to-tuple](#converting-a-struct-to-a-tuple) example from above. It was demonstrated using a single splice, but it would be simpler if we had a range splice:
 
 ::: cmptable
-### With Single Splice
+#### With Single Splice
```

Comment:
No 3 `#`s is correct, that's what `cmptable` is for as a heading.

Reply:
Thanks, done.